internal/action/commands.go

```diff
@@ -18,6 +18,10 @@ func ShowFlags() []cli.Flag {
 			Aliases: []string{"y"},
 			Usage:   "Always answer yes to yes/no questions",
 		},
+		&cli.BoolFlag{
```

Comment:
Rather than a flag this should be a config option, I think.

Reply:
Good point. For edit for example I think we shouldn't replace the reference. Otherwise that's an confusing user experience, isn't it?